wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/monitoring/ClusterTopologyMonitorImpl.java

```diff
@@ -1037,5 +1058,11 @@ private void readerThreadFetchTopology(final Connection connection, final @Nulla
         LOGGER.fine(Utils.logTopology(hosts));
       }
     }
+
+    private long calculateBackoffWithJitter(int attempt) {
+      long backoff = INITIAL_BACKOFF_MS * Math.round(Math.pow(2, Math.min(attempt, 6)));
+      backoff = Math.min(backoff, MAX_BACKOFF_MS);
+      return Math.round(backoff * (0.5 + random.nextDouble() * 0.5));
```

Comment:
for my own knowledge, what is the benefit of using a random factor in this calculation?